modules/auxiliary/admin/http/epmp1000_get_chart_cmd_exec.rb

```diff
@@ -0,0 +1,116 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
```

Comment:
How is this different than modules/auxiliary/admin/http/epmp1000_get_chart_cmd_exec.rb?  These seem to exploit the same vulnerability.  As stated elsewhere, if you can execute commands, you can likely get a shell, therefore this should be an exploit module.

Reply:
Do you mean _cmd_shell.rb module? Yes, they exploit the same vuln. But it seemed appropriate to build both. _cmd_exec.rb can run one set of command(s) against multiple hosts, & _cmd_shell.rb exploit will get one shell at a time. I think we talked about this over one of our slack calls..